broflake/broflake.go

```diff
@@ -8,6 +8,7 @@ import (
 )
 
 func Wrap(ll net.Listener, certPEM string, keyPEM string) (net.Listener, error) {
-	// TODO: update the Broflake library to accept cert and key as PEM encoded strings
+	// TODO (allan): update the Broflake library to accept cert and key as PEM encoded strings
+	// Maybe it's already done. Investigate!
```

Comment:
[It does](https://github.com/getlantern/browsersunbounded/blob/b0693659265d38862b56f96e17a31173012403e6/egress/egresslib.go#L284C20-L284C31) ([ref](https://pkg.go.dev/crypto/tls#X509KeyPair)).